src/main/java/CMV.java

```diff
@@ -185,4 +198,58 @@ private boolean LIC14() {
     // TODO: Implementation.
     return false;
   }
+
+  private boolean insideCircle(Point a, Point b, Point c, double radius){
```

Comment:
I'm unsure about this function. I know we said we shouldn't point out if there are "nicer" solutions, but there already exists a function `isInsideCircle` in #26, and an example of how that function can be used to calculate if three points can be contained in the same circle.

Reply:
IDK, change it or not I will approve it. Just to get it merged